api/src/pcapi/routes/adage_iframe/matomo.py

```diff
@@ -0,0 +1,33 @@
+"""
+    experimental module to integrate matomo tracking in passculture api
+"""
+import functools
+import hashlib
+import typing
+
+import requests
```

Comment:
Il est préférable d'utiliser notre wrapper (qui logge, définit un timeout, etc.) : 

```suggestion
from pcapi.utils import requests
```